convert-redis-to-gemfire/session-state/spring-session-boot-json/redis/build.gradle

```diff
@@ -4,13 +4,13 @@
  */
 
 plugins {
-	id 'org.springframework.boot' version '2.2.1.RELEASE'
+	id 'org.springframework.boot' version '3.1.3'
 	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
```

Comment:
I think we need to upgrade this. Was done in another project, we should do the same here

Reply:
Thanks, I missed this one.